libs/remix-ui/run-tab/src/lib/actions/events.ts

```diff
@@ -14,6 +14,10 @@ import { shortenAddress } from "@remix-ui/helper"
 const _paq = window._paq = window._paq || []
 
 export const setupEvents = (plugin: RunTab, dispatch: React.Dispatch<any>) => {
+  const currentNetwork = {
```

Comment:
could you add a comment in a code on why you need to do this?

Reply:
done